lib/internal/bootstrap/pre_execution.js

```diff
@@ -245,6 +245,16 @@ function setupFetch() {
         throw new ERR_WEBASSEMBLY_RESPONSE('body has already been used');
       }
 
+      const handler = require('internal/v8').getWasmStreamingHandler();
+      const cached = await handler?.get(response.url);
```

Comment:
As I understand this, this will only work when loading WebAssembly from a remote origin, and only if the URL is stable, and only if the WebAssembly module does not change.